ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -18,6 +18,22 @@
 
 ### Single Zone Units ###
 
+AHU_ECON_HWSC_ETM_SFM_FDPSM_SFVSC_EFVSC_SDM_EDM:
+  guid: "06e2e862-e9c2-46ec-91cd-d4b43cb44449"
+  description: "Single zone AHU with supply and return fans, with supply heating from run around coil."
```

Comment:
@Ciaran-Oflannagain just noticed this, if this is a single zone unit you need to use `discharge` instead of `supply` and use the appropriate abstracts (ECOND, HWDC, etc.)